.github/workflows/default.yml

```diff
@@ -7,5 +7,13 @@ jobs:
     - uses: actions/checkout@v2
     - name: Install dependencies
       run: lein deps
+    - name: Start PG container
+      run: docker run -d -p 9091:9091 prom/pushgateway
```

Comment:
Hey, GitHub Actions can manage services for you. It'll make your life easier.

https://github.com/actions/example-services/tree/main/.github/workflows